src/main/docker/bin/run_nginx

```diff
@@ -1,2 +1,10 @@
 #!/bin/sh
-exec nginx -g "daemon off;"
+if [[ -z "${NO_PROCESSING}" ]]
+then
+   PROCESSED_CONF=$(mktemp)
+   envsubst < /etc/nginx/nginx.conf > $PROCESSED_CONF && \
+   mv $PROCESSED_CONF /etc/nginx/nginx.conf && \
+   exec nginx -g 'daemon off;'
```

Comment:
Flytte utenfor conditional for å unngå repitisjon?

Reply:
Den bør snus.... Aner ikke hva jeg tenkte her.

Tanken er at man skal kunne skippe prosessering av filen om man ikke ber om det, siden det fordrer at rettigheter er satt korrekt på filen slik at man kan skrive over den.

Men det er kanskje like greit å bare droppe conditional.. Hvis det feiler, så feiler det iallefall konsistent. Det er nesten bedre enn en conditional.